app/books/page.jsx

```diff
@@ -1,18 +1,162 @@
-import Search from "./Search";
```

Comment:
I divided the code into components to reduce the length of code in one file. Please don't make changes.

Reply:
Do you really think the search needs a separate file? I don't think so ..I already worked here..also It was very hard for me to work on it..

Reply:
And If you know I work on the book page previously..so it's my task to update on it..

Reply:
ok , let me check... are you using the search bar everywhere? @s2sharpit

Reply:
done git pull but I can't see any tailwind CSS there? @s2sharpit 
![Screenshot (503)](https://github.com/rohansx/informatician/assets/106610322/455ff7ee-a838-4535-9c4c-8fdd4f1f54d4)

Reply:
This is the screenshot from here: https://github.com/rohansx/informatician/blob/next-ts-tw-migrate/app/books/page.jsx
In this the tailwindcss is implemented. @crocmons 

![image](https://github.com/rohansx/informatician/assets/104740281/c52238bd-9b84-4a47-b81e-2ef1ed68f0b8)

Reply:
ok, I will fix this in a moment ..don't worry about it.. @s2sharpit